releases/cockroachcloud-02022020.md

```diff
@@ -0,0 +1,42 @@
+---
+title: What&#39;s New in CockroachCloud
+toc: true
+summary: Additions and changes in CockroachCloud.
+build_for: [cockroachcloud]
+---
+
+## March 02, 2020
+
+In addition to various updates, enhancements, and bug fixes, this beta release includes the following major highlights:
+
+- CockroachCloud clusters running CockroachDB v19.2 have been upgraded to [v19.2.4](https://www.cockroachlabs.com/docs/releases/v19.2.4.html). All new clusters will now be created with CockroachDB v19.2.4.
+- CockroachCloud now offers two options for per-node hardware configuration instead of three options. The hardware configuration [pricing](../stable/cockroachcloud-create-your-cluster.html#step-2-select-the-cloud-provider) has been updated accordingly.
+
+### Security updates
+
+- CockroachCloud now requires that the password for a SQL user is at least 12 characters in length.
+- CockroachCloud now allows you to download the cluster's CA certificate directly from the shell instead of restricting the download functionality to a web browser.
+
+### General changes
+
+- Added a **Sign up** link to the [CockroachCloud **Log In** page](https://cockroachlabs.cloud/).
+- While [creating a new cluster](../stable/cockroachcloud-create-your-cluster.html), you can now type in the number of nodes you want in the cluster instead of having to click the `+` sign repeatedly.
+- The [**Create cluster**](../stable/cockroachcloud-create-your-cluster.html) page now displays the estimated hourly cost instead of the monthly cost.
+- Removed the cluster creation banner displayed at the top of the [**Clusters page**](cockroachcloud-cluster-management.html#view-clusters-page).
```

Comment:
This link is broken. Use:

../v19.2/cockroachcloud-cluster-management.html#view-clusters-page